test/e2e-tests.sh

```diff
@@ -62,10 +62,15 @@ export REPO_URL="https://github.com/ncskier/go-hello-world"
 export EXPECTED_RETURN_VALUE="Hello World!"
 export REGISTRY="gcr.io/${E2E_PROJECT_ID}/${E2E_BASE_NAME}-e2e-img"
 export TEKTON_PROXY_URL="http://localhost:9097/proxy/apis/tekton.dev/v1alpha1/namespaces/tekton-pipelines"
+export CSRF_HEADERS_STORE="csrf_headers.txt"
 
 # Kubectl static resources
 kubectl apply -f ${tekton_repo_dir}/test/resources/static
 
+curl -D $CSRF_HEADERS_STORE http://localhost:9097/v1/token
+export CSRF_TOKEN=`grep 'X-Csrf-Token' $CSRF_HEADERS_STORE | sed -e 's/^X-Csrf-Token: //;s/\r//'`
```

Comment:
Is the casing here correct/required? It's X-CSRF-Token elsewhere

Reply:
yeah header case gets mangled all sorts of ways, to avoid confusion I think I'll change this to `grep -i 'X-CSRF-Token'` for a case-insensitive match